plugins/discourse-narrative-bot/plugin.rb

```diff
@@ -97,7 +97,11 @@ def generate
           raise Discourse::InvalidParameters.new("#{key} must be present") unless params[key]&.present?
         end
 
-        rate_limiter = RateLimiter.new(current_user, 'svg_certificate', 3, 1.minute)
+        if params[:user_id].to_i != current_user.id
+          rate_limiter = RateLimiter.new(current_user, 'svg_certificate', 3, 1.minute)
```

Comment:
>  the ratelimit needs to be at least 4 burst (2 requests twice)

I'm not too familiar with RateLimiter and its lua-ness 😉 Does max=3 and secs=1.minute satisfy this requirement or should it be max=4?

Reply:
I kept that at the old value, because non staff users viewing others' completion certificates should be rare. Only being more lenient for your own. This is also what anonymous users get. I will admit it's not backed by data.